interface/src/app/analyze/analysis/analysisDetail.tpl.html

```diff
@@ -1,11 +1,20 @@
 <!--
   TODO each sample should be built using a reusable Angular directive
 -->
-<h1>Samples
-  <small>(drag <i class="fa fa-bars" aria-hidden="true"></i> to reorder,
-    click <i class="fa fa-times" aria-hidden="true"></i> to delete,
-    choose <input type="radio" disabled> to set group)</small>
-</h1>
+
+<div class="page-header">
+  <h1>Samples
+    <small>(drag <i class="fa fa-bars" aria-hidden="true"></i> to reorder,
+      click <i class="fa fa-times" aria-hidden="true"></i> to delete,
+      choose <input type="radio" disabled> to set group)</small>
+  </h1>
+  <ml-model-view></ml-model-view>
+  <ml-model-validator model-id="modelInUrl" is-valid-model="isValidModel">
+  </ml-model-validator>
+</div>
+
+<div ng-if="isValidModel">
```

Comment:
@dongbohu - tag here is @cgreene / @casey bugs someone else ;)

I'm imagining a situation where a user searches and selects datasets from multiple organisms. How will we show those results? I can't think of a sensible way to do that. Restricting by `mlmodel` first prevents us from getting stuck in a situation where the user is selecting multiple experiments that we can't sensibly show together (if we facet the search results by the `mlmodel`'s `organism`. Though it's not a problem with one organism in the production instance, it will be. Seems good to handle this now.